test/plugin/test_out_google_cloud_grpc.rb

```diff
@@ -155,6 +155,21 @@ def test_struct_payload_non_utf8_log
     end
   end
 
+  # For grpc path, the log name is sent as a part of the log entry, thus can be
+  # verified directly.
+  def test_tag_acceptance
+    setup_logging_stubs_block = lambda do |_converted_tag, setup_driver_block|
+      setup_logging_stubs do
```

Comment:
Right. Just added equivalent verification to the grpc side. Thank god I can get rid of those messy lambda definitions now.